src/engraving/rendering/dev/chordlayout.cpp

```diff
@@ -1429,6 +1444,10 @@ void ChordLayout::layoutChords1(LayoutContext& ctx, Segment* segment, staff_idx_
     const track_idx_t partStartTrack = part ? part->startTrack() : startTrack;
     const track_idx_t partEndTrack = part ? part->endTrack() : endTrack;
 
+    if (isTab) {
+        skipAccidentals(segment, startTrack, endTrack);
```

Comment:
And where is `acc->setSkipDraw(true);` set back in `acc->setSkipDraw(false);`?

Reply:
Good point. I'll add it